clients.json

```
@@ -1772,6 +1772,16 @@
     "active": true
   },
 
+  {
+    "name": "NewLife.Redis",
+    "language": "C#",
+    "url": "https://github.com/NewLifeX/NewLife.Redis",
+    "repository": "https://github.com/NewLifeX/NewLife.Redis",
+    "description": "The high-performance redis client supports .NETCORE/.NET4.0/.NET4.5, which is specially optimized for big data and message queuing. The average daily call volume of single online application is 10 billion",
+    "authors": ["BigStone"],
```

This should contain the Twitter handles of the author(s). It appears that the handle provided is inactive so you may just want to leave this empty like so.

```suggestion
    "authors": [],
```